CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master
```

Comment:
You’ll probably want to rebase onto master, because the changelog entries need to go in a new master section, not v0.24.0.